index.js

```diff
@@ -50,4 +50,17 @@ types.forEach(function(type) {
   normalize[type] = normalize.bind(null, type);
 });
 
+normalize.date = function(value) {
+  var args = [value].concat(Array.prototype.slice.call(arguments, 1));
+  var asNumber = normalize.number.apply(null, args);
```

Comment:
instead of separate calls, maybe we should just have a custom predicate function that checks for number or date.